dask_jobqueue/lsf.py

```diff
@@ -196,3 +150,49 @@ def lsf_detect_units():
             "default unit of %s." % unit
         )
     return unit
+
+
+class LSFCluster(JobQueueCluster):
+    __doc__ = """ Launch Dask on a LSF cluster
+
+    Parameters
+    ----------
+    queue : str
+        Destination queue for each worker job. Passed to `#BSUB -q` option.
+    project : str
+        Accounting string associated with each worker job. Passed to
+        `#BSUB -P` option.
+    {job}
+    ncpus : int
+        Number of cpus. Passed to `#BSUB -n` option.
+    mem : int
+        Request memory in bytes. Passed to `#BSUB -M` option.
+    walltime : str
+        Walltime for each worker job in HH:MM. Passed to `#BSUB -W` option.
+    {cluster}
+    job_extra : list
+        List of other LSF options, for example -u. Each option will be
+        prepended with the #LSF prefix.
+    lsf_units : str
+        Unit system for large units in resource usage set by the
+        LSF_UNIT_FOR_LIMITS in the lsf.conf file of a cluster.
+
+    Examples
+    --------
+    >>> from dask_jobqueue import LSFCluster
+    >>> cluster = LSFCluster(queue='general', project='DaskonLSF',
+    ...                      cores=15, memory='25GB')
+    >>> cluster.scale(10)  # this may take a few seconds to launch
```

Comment:
It would be great if all the docstring had the same example. Probably using `.scale(jobs=...)` and `.adapt(max_jobs=...)` is the thing that makes the most sense for `dask-jobqueue` users.

Can be done after this PR is merged.

Reply:
Fixed.